force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +227,23 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
+        String helloWorldFinal ='';
         // Use a for loop to append the inputString to the result string repeatCount times
         /*
         for () {
 
             // If it is not the last iteration, add a semicolon to separate the strings
         }
         */
+        for (Integer i = 0; i < repeatCount; i++){
+            helloWorldFinal += inputString;
+            if (i < repeatCount - 1) {
+                helloWorldFinal += '; ';
+            }
+        }
 
```

Comment:
Good. Take a look at the String method removeEnd() makes you life easier for this problem.